spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfiguration.java

```diff
@@ -88,6 +88,9 @@
 	@Value("${management.port:${MANAGEMENT_PORT:${server.port:${SERVER_PORT:${PORT:8080}}}}}")
 	private int managementPort;
 
+	@Value("${management.contextPath:${management.context-path:${MANAGEMENT_CONTEXT_PATH:${server.contextPath:${SERVER_CONTEXT_PATH:${CONTEXT_PATH:}}}}}}")
```

Comment:
We shouldn't add more of these. We should use the `RelaxedPropertyResolver ` already available.